events/developer-meetups.mdx

```diff
@@ -0,0 +1,70 @@
+---
+id: developer-meetups
+title: DHIS2 Developer Meetups
+---
+
+import NextSecondThursday from '@site/src/nextSecondThursday';
+
+Did you know that the DHIS2 Developer Community hosts monthly meetups?
+
+## What are these meetups?
+
+A series of online and informal meetings with the DHIS2 Developer Community. These meetups are a great way for developers working on DHIS2-related projects to connect, ask questions and stay updated.
+
+Topics range from meet and greet to technical web development discussions. These are some of topics covered so far in past meetings:
+
+-   DHIS2 UI library and Design System
+-   What makes a good generic DHIS2 application?
+-   How to use the DataStore?
+-   DHIS2 application testing with Cypress
+-   Meet and Greet, general Q&A
+
+## Next Meetup Time & Date
+The DHIS2 Developer Meetup is happening every second Thursday of the month.
+
+The next Developer Meetup is on ***<NextSecondThursday />\* at 11:00 CE(S)T\*\*.***
```

Comment:
I think it's easier to just say it's the second Thursday of the month :D 

(I also think this could potentialyl cause issues with docusaurus hydration if the statically generated time differs from the client side)

Reply:
The date is properly rendered based on client-side time, so should not cause an issue here!